test/cypress/dashboards/integration/layout.cy.js

```diff
@@ -144,3 +144,21 @@ describe('Chart synchronized series state', () => {
     });
 
 });
+
+describe('Components in multiple layouts', () => {
+    before(() => {
+        cy.visit('/dashboards/cypress/duplicated-ids');
+    })
+
+    it('Components should be placed in different dashboards', () => {
+        cy.window().its('Dashboards.boards').should('have.length.gte', 1).then(boards => {
```

Comment:
Isn't it safer to check if the length is 2? Isn't it what we expect here? 
The `gte` is a bit confusing

Reply:
I can simplify it.